### DISCUSSION TITLE
PackFileCacheStrategy/FileSystemInfo warnings in console when using buildDependencies

### DISCUSSION THREAD

**Discussion by GaganpreetKaurKalsi** [webpack/webpack#17913] [Q&A]
# Bug report

**What is the current behavior?**

I am getting below warnings in console on doing production build. I am using filesystem cache and importing a few files in my webpack.config.js using relative path.

```
[webpack.cache.PackFileCacheStrategy/webpack.FileSystemInfo] 
Resolving 'build-config/alias.config' in 
/Users/gaganpreetkaurkalsi/web-app/build-config/webpack 
for build dependencies doesn't lead to expected result 
'/Users/gaganpreetkaurkalsi/web-app/build-config/alias.config.js',
but to 'Error: Can't resolve 'build-config/alias.config' in 
'/Users/gaganpreetkaurkalsi/web-app/build-config/webpack'' instead. 
Resolving dependencies are ignored for this path.
```

**If the current behavior is a bug, please provide the steps to reproduce.**
Using cache in production mode should enable you to see these warnings.

```
cache: {
    name: "prod",
    type: "filesystem",
    allowCollectingMemory: true,
    buildDependencies: {
      config: [__filename],
    },
  },
```

**What is the expected behavior?**
Any work around this warning? Any solution?

**Other relevant information:**
webpack version:  5.89.0
Node.js version: 14
Operating System: MacOS
Additional tools:

**Comment by snitin315** [comment]
@GaganpreetKaurKalsi Can you provide a reproducible GitHub repo?
